src/Security/Authorization/Core/src/Microsoft.AspNetCore.Authorization.csproj

```diff
@@ -1,4 +1,4 @@
-﻿<Project Sdk="Microsoft.NET.Sdk">
+<Project Sdk="Microsoft.NET.Sdk">
```

Comment:
Can we change this assembly to be `netstandard2.0`? That will be necessary for it to work on WebAssembly.

If you want that to be a separate change that I do later, that's OK, but it would be handy if you could check that your refactoring will be compatible with this change.

Reply:
Yep will make that change now